src/irrational.jl

```diff
@@ -0,0 +1,70 @@
+type IrrationalContinuedFraction{T<:Integer, C} <: ContinuedFraction{T}
+    precision::Int
+    quotients::Vector{T}
+end
+
+eps_error(c::MathConst) = 2<<20 # assume accurate to within 20 bits?
```

Comment:
Nitpick: `2 << 20 == 1 << 21`, I assume you meant `1 << 20`? Not really relevant as that one bit doesn't make a lot of difference :)